code/Java/Hw.java

```diff
@@ -0,0 +1,12 @@
+import java.io.*;
+import java.util.*;
+
+
+class HelloWorld
+{
```

Comment:
Looks like you are going about this workflow perfectly. However I strongly recommend using [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) for programming in Java. In bigger organisations, code maintenance is as important as adding newer modules.
Congrats on your very first PR 😃!